src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -40,32 +40,30 @@ public class PersonCard extends UiPart<Region> {
     private Label email;
     @FXML
     private FlowPane tags;
+    @FXML
+    private Label header;
 
     /**
      * Creates a {@code PersonCode} with the given {@code Patient} and index to display.
      */
-    public PersonCard(Patient person, int displayedIndex) {
-        this(person, displayedIndex, false);
-    }
-
-    /**
-     * Creates a {@code PersonCode} with the given {@code Patient} and its associated task list and index to display.
-     */
-    public PersonCard(Patient person, int displayedIndex, boolean taskListFlag) {
+    public PersonCard(Patient patient, int displayedIndex) {
         super(FXML);
-        this.person = person;
+        cardPane.setSpacing(1);
+        cardPane.setStyle("-fx-padding: 1;" + "-fx-border-style: solid inside;"
+                + "-fx-border-width: 1;" + "-fx-border-insets: 1;"
+                + "-fx-border-radius: 2;" + "-fx-border-color: black;");
+        this.patient = patient;
         id.setText(displayedIndex + ". ");
-        name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
-        person.getTags().stream()
+        name.setText(patient.getName().fullName);
+        phone.setText(patient.getPhone().value);
+        address.setText(patient.getAddress().value);
+        email.setText(patient.getEmail().value);
+        patient.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
```

Comment:
Minor issue regarding this but you don't need to change this as I realized that this is a remnant of the original codebase. I think that the values should not be accessed directly to set the text for the labels. I will create a separate issue to resolve this violation.